resources/qml/WelcomePages/WelcomeContent.qml

```diff
@@ -7,56 +7,76 @@ import QtQuick.Controls 2.3
 import UM 1.3 as UM
 import Cura 1.1 as Cura
 
-
 //
 // This component contains the content for the "Welcome" page of the welcome on-boarding process.
 //
 Item
 {
     UM.I18nCatalog { id: catalog; name: "cura" }
 
-    Column  // Arrange the items vertically and put everything in the center
+    // Arrange the items vertically and put everything in the center
+    Column
     {
-        anchors.centerIn: parent
-        width: parent.width
-        spacing: 2 * UM.Theme.getSize("wide_margin").height
-
-        Label
-        {
-            id: titleLabel
-            anchors.horizontalCenter: parent.horizontalCenter
-            horizontalAlignment: Text.AlignHCenter
-            text: catalog.i18nc("@label", "Welcome to Ultimaker Cura")
-            color: UM.Theme.getColor("primary_button")
-            font: UM.Theme.getFont("huge")
-            renderType: Text.NativeRendering
-        }
+        anchors.horizontalCenter: parent.horizontalCenter
+        anchors.verticalCenter: parent.verticalCenter
+        spacing: UM.Theme.getSize("thick_margin").height
 
         Image
         {
             id: curaImage
             anchors.horizontalCenter: parent.horizontalCenter
             source: UM.Theme.getImage("first_run_welcome_cura")
+            scale: 0.666
```

Comment:
(Same issue as above: This image doesn't get scaled with the screen scale factor and its width depends on the source image.)